repositories/strategies/duplicateStrategy.js

```diff
@@ -226,22 +277,37 @@ const duplicateSectionStrategy = async (
     sectionId: section.id
   });
 
-  await Promise.all(
-    pagesToDuplicate.map(({ position, ...page }) =>
-      duplicatePageStrategy(trx, page, position, {
+  for (let i = 0; i < pagesToDuplicate.length; ++i) {
+    const { position, ...page } = pagesToDuplicate[i];
+    await duplicatePageStrategy(
```

Comment:
Do we need to await here?

Reply:
Pages can reference answers in previous pages so we need them to be created before we carry on.